src/parachain/redeem.ts

```diff
@@ -51,10 +62,10 @@ export interface RedeemAPI extends TransactionAPI {
     request(
         amount: MonetaryAmount<WrappedCurrency, BitcoinUnit>,
         btcAddressEnc: string,
-        vaultId?: AccountId,
+        vaultId?: InterbtcPrimitivesVaultId,
         atomic?: boolean,
         retries?: number,
-        availableVaults?: Map<AccountId, MonetaryAmount<WrappedCurrency, BitcoinUnit>>
+        availableVaults?: Map<string, MonetaryAmount<WrappedCurrency, BitcoinUnit>>
```

Comment:
Similar to issue, not sure we would want to have only one option: submit an array or the vaultId.